client/sites/staging-site/components/staging-site-card/card-content/staging-site-creation-card/staging-site-creation-card-content.scss

```diff
@@ -0,0 +1,69 @@
+@import "@wordpress/base-styles/mixins";
+
+.staging-site-creation-card {
+	padding: 2rem;
+	display: flex;
+	justify-content: space-between;
+	width: 100%;
+	max-width: 100%;
+	box-sizing: border-box;
+	gap: 1rem;
+	position: relative;
+	overflow: hidden;
+	background-image: url( ./site-creation-graphic.svg );
+	background-repeat: no-repeat;
+	background-position: center right;
+	background-size: auto 100%;
+
+	@media screen and ( max-width: 1024px ) {
```

Comment:
Nit: Maybe you can use any of the existing [breakpoints](https://github.com/WordPress/gutenberg/blob/48b43b2de9b93adc2ec872a78140d4b7a79e823c/packages/base-styles/_breakpoints.scss#L9) or the imported [mixins](https://github.com/WordPress/gutenberg/blob/48b43b2de9b93adc2ec872a78140d4b7a79e823c/packages/base-styles/_mixins.scss#L105) for this. But not a hard requirement if it does not fit well for any of those 🙂